caput/time.py

```diff
@@ -566,7 +566,7 @@ def _fixup_interval_and_step(self, t0, t1, step):
         # Get the ends of the search interval
         t0 = ensure_unix(t0)
         if t1 is None:
-            t1 = t0 + 24 * 3600.0
+            t1 = t0 + 24 * 3600.0 * SIDEREAL_S
```

Comment:
This probably should be STELLAR_S

Reply:
Otherwise, I think you make a fair point!

Reply:
back to wikipedia I go...

Reply:
Do you agree it should be stellar day in the end?

Reply:
I think so. If we've specified RA and Dec in ICRS coordinates we should use that reference to calculate the next transit. I could test it by comparing the predicted time between the next two transits to the stellar/diereal day. Practically, it's only an 8ms difference.